docker/index.js

```diff
@@ -134,109 +142,51 @@ app.post('/trigger', async (req, res) => {
 		},
 	}
 
-	try{
+	try {
 		// upload metadata to ipfs
 		uploadMetadataRes = await pinata.pinJSONToIPFS(metadata, metadataOption);
 	}
-	catch (e){
+	catch (e) {
 		// if fail upload metadata, uploaded image is unpined in pinata.
-		console.error(e);
+		console.error('Fail ipfs upload', e);
 		await pinata.unpin(uploadImgRes.IpfsHash);
 		await ain.db.ref(errorPath).setValue({
 			value: {
-				state:"Error",
-				msg:"Metadata upload fail. check your inforamtion of metadata"
+				state: "Error",
+				msg: "Metadata upload fail. check your inforamtion of metadata"
 			},
 		})
-		.catch((e) => {
-			console.error(`setValue failure:`, e);
-		});
+			.catch((e) => {
+				console.error(`setValue failure:`, e);
+				return;
+			});
 
 		return;
 	}
 
-	const ainRes = await ain.db.ref(outputPath).setValue({
+	await ain.db.ref(outputPath).setValue({
+		nonce: -1,
+		gas_price: 500,
 		value: {
-			contract:{
-				network:value.contract_info.network,
-				chain_id:value.contract_info.chain_id,
-				account:value.contract_info.account,
-				token_id:value.contract_info.token_id,
+			contract: {
+				network: value.contract.network,
+				chain_id: value.contract.chain_id,
+				account: value.contract.account,
+				token_id: value.contract.token_id,
 				new_metadata: uploadMetadataRes.IpfsHash,
 			},
 			verified_at: Date.now(),
 			trigger_verification_account: BOT_ADDRESS,
 		},
-	}).catch((e) => {
+	})
+	.catch((e) => {
```

Comment:
Even if an error occurs, the success log at the bottom line seems to be executed.
because of promise chaining.